pygmt/base_plotting.py

```diff
@@ -1982,3 +1982,218 @@ def update_pointers(data_pointers):
             with file_context as fname:
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("meca", arg_str)
+
+    @fmt_docstring
+    @use_alias(
+        A="sector_width",
+        B="frame",
+        C="cmap",
+        D="shift_sectors",
+        Em="plot_vectors",
+        F="no_scalebar",
+        G="color",
+        I="inquire",
+        JX="diameter",
+        L="labels",
+        M="vector_params",
+        Q="conf_level",
+        R="region",
+        S="norm",
+        T="orient_data",
+        U="timestamp",
+        V="verbose",
+        W="pen",
+        X="xshift",
+        Y="yshift",
+        Z="scale",
+        i="columns",
+        p="perspective",
+        t="transparency",
+    )
+    @kwargs_to_strings(R="sequence", i="sequence_comma", p="sequence")
+    def rose(self, length=None, azimuth=None, data=None, **kwargs):
+        """
+        Plot windrose diagrams or polar histograms.
+
+        Takes a matrix, (length,azimuth) pairs, or a file name as input
+        and plots windrose diagrams or polar histograms (sector diagram
+        or rose diagram).
+
+        Must provide either *data* or *length* and *azimuth*.
+
+        Options include full circle and half circle plots. The outline
+        of the windrose is drawn with the same color as *MAP_DEFAULT_PEN*
+        (see: gmt-docs:`gmt.conf.html#term-MAP_DEFAULT_PEN`).
```

Comment:
```suggestion
        (see :gmt-docs:`gmt.conf.html#term-MAP_DEFAULT_PEN`).
```

This doesn't render as a link once Sphinx builds it. I think this should fix it.

Reply:
thanks for the improvement @willschlitzer